dashboard/lib/policies/lti.rb

```diff
@@ -69,13 +69,19 @@ module AccessTokenScopes
 
   MAX_COURSE_MEMBERSHIP = 650
 
-  def self.get_account_type(id_token)
-    id_token[LTI_ROLES_KEY].each do |role|
+  def self.get_account_type(roles)
```

Comment:
You're probably working on it (as this is a draft) but doing a search in the repo, this function gets called twice in `lti_test.rb`, so you'll want to update the tests with the new input for this function.

Reply:
Ah yes, sorry. Missed that usage down in `services/lti.rb`. This change looks good to me 👍